bears/elm/ElmLintBear.py

```diff
@@ -0,0 +1,38 @@
+from shutil import which
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='elm-format',
+        use_stdin=True,
+        use_stderr=True,
+        use_stdout=False,
+        output_format='regex',
+        output_regex='(?P<line>\d+)│(?P<message>[\s\S]*)')
```

Comment:
`elm-format` reports only one error at a time, so that will not be a problem IIUC. Here is a use case: 
```
 yash  ~  elm-format --stdin
l =
  ["one", "two", "three"
a, b =
  "Hello World!"
ERRORS
-- SYNTAX PROBLEM ------------------------------------------------------ <STDIN>

I ran into something unexpected when parsing your code!

3│ a, b =
   ^
I am looking for one of the following things:

    ","
    "]"
    whitespace


 yash  ~  

```
CC @Makman2